fighters/common/src/function_hooks/init_settings.rs

```diff
@@ -109,7 +109,11 @@ unsafe fn init_settings_hook(boma: &mut BattleObjectModuleAccessor, situation: s
         // Set GroundCliffCheckKind here to pass into init_settings
 
         if (boma.kind() == *FIGHTER_KIND_RYU || boma.kind() == *FIGHTER_KIND_KEN)
-            && boma.is_status_one_of(&[*FIGHTER_STATUS_KIND_SPECIAL_S, *FIGHTER_RYU_STATUS_KIND_SPECIAL_S_COMMAND, *FIGHTER_RYU_STATUS_KIND_SPECIAL_S_LOOP, *FIGHTER_RYU_STATUS_KIND_SPECIAL_S_END]) {
+            && boma.is_status_one_of(&[
+                *FIGHTER_STATUS_KIND_SPECIAL_S,
+                *FIGHTER_RYU_STATUS_KIND_SPECIAL_S_COMMAND,
+                *FIGHTER_RYU_STATUS_KIND_SPECIAL_S_LOOP,
+                *FIGHTER_RYU_STATUS_KIND_SPECIAL_S_END]) {
             cliff_check_kind = app::GroundCliffCheckKind(*GROUND_CLIFF_CHECK_KIND_ON_DROP_BOTH_SIDES);
```

Comment:
I'm not going to make you go back and change it, but in the future move the `{` to the next line as it makes it neater/easier to look at (it is very easy to see the `cliff_check_kind` line as part of the conditional)